.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,65 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Create a temporary Git tag"
+        run: git tag "$MARKETING_VERSION"
```

Comment:
We need to extract the MARKETING_VERSION before this step now.

Reply:
That was a silly mistake. Fixed in 388781e5714fee0f72d01718e719608cc83e8e8a.